pkgs/by-name/fr/frequest/package.nix

```diff
@@ -33,6 +33,11 @@ stdenv.mkDerivation (finalAttrs: {
   ];
   sourceRoot = "frequest";
 
+  patches = [
+    # TODO: Remove once a version is delivered with https://github.com/fabiobento512/FRequest/commit/22e98970b78f6d9ca0037c39c2492e93e836abff.
+    ./0001-Change-desktop-icon-name.patch
```

Comment:
Could you please `fetchpatch` it ? It would ensure that no one will forget to also remove the patch file itself from nixpkgs, and it would demand less space in nixpkgs clones.

Reply:
Thank you, should be OK now.